modules/ocf/templates/broker/broker.conf.erb

```diff
@@ -0,0 +1,3 @@
+[broker]
+host = hozer-65.ocf.berkeley.edu
```

Comment:
likewise don't hardcode these things, add a cname to the new vm for `broker.ocf.berkeley.edu` or something like that

Reply:
I'd just put `broker` here instead of the FQDN too, since that's simpler and a bit cleaner.

Reply:
I already added a CNAME for it, I just blanked here :P